rest/config.go

```diff
@@ -121,7 +121,7 @@ type DbConfig struct {
 	Users                        map[string]*db.PrincipalConfig `json:"users,omitempty"`                        // Initial user accounts
 	Roles                        map[string]*db.PrincipalConfig `json:"roles,omitempty"`                        // Initial roles
 	RevsLimit                    *uint32                        `json:"revs_limit,omitempty"`                   // Max depth a document's revision tree can grow to
-	ImportDocs                   interface{}                    `json:"import_docs,omitempty"`                  // false, true, or "continuous"
+	ImportDocs                   interface{}                    `json:"import_docs,omitempty"`                  // The only valid value if this is provided is "continuous".  Xattrs must be enabled.
```

Comment:
Any reason not to change to a `*string`?

I didn't try that in the first pass just to keep the PR as minimal of a change as possible, but since the only valid values are now `nil` and "continuous", maybe it makes sense.  Thoughts welcome.

Reply:
Ideally we'd simplify it to boolean at this point - 'continuous' is really just a relic of the old options.  We probably need to continue to support 'continuous' for backward compatibility, but I think we should also treat true==continuous and false==nil.

Reply:
Fixed